### PR TITLE
modify args type for handler.

### PR DESCRIPTION
**Pull request by sassy**
modify to use any type of handler.

argument function depends on express, so specify Express.Request/Express.Response type.

### CODE REVIEW

**Comment by sassy** [comment]
@navarone-feekery  
What do you think it?

File: src/client.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import * as Express from 'express';
```
Comment (uich) [inline src/client.ts:1]: Please modify `Express` to `express`.

Comment (sassy) [reply src/client.ts]: I fix it.

**Comment by sassy** [comment]
@uich Thank you review it.
I got it.

**Comment by sassy** [comment]
Thank you for merging it.

**Pull request merged**